seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment3/ExcelSheetRead.java

```diff
@@ -0,0 +1,49 @@
+package seleniumPractise.prachiJ.Assingment3;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public class ExcelSheetRead {
+	Workbook workbook = null;
+
+	ExcelSheetRead(String path) throws IOException {
+		File file = new File(path);
+		FileInputStream input = new FileInputStream(file);
+		workbook = new XSSFWorkbook(input);
+	}
+
+	Object[][] getRowData(String sheetName) throws IOException {
+		int FirstRow = workbook.getSheet(sheetName).getFirstRowNum();
+		int FirstCol = workbook.getSheet(sheetName).getRow(0).getFirstCellNum();
+		int rows = workbook.getSheet(sheetName).getLastRowNum(); // 12 -
+																	// excluding
+																	// header
+		int cols = workbook.getSheet(sheetName).getRow(0).getLastCellNum(); // 10
+		Object[][] cellData = new String[rows][cols]; // 0 to 11 ka array for
+														// rows
+		boolean isHeader = true;
```

Comment:
Hi Prachi,

"isHeader" variable value, you are explicitly giving in the getRowData() method.
It should be passed as a parameter when method is called.